src/main/java/core/basesyntax/strategy/SupplyHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class SupplyHandler implements OperationHandler {
+    private static final String SUPPLY = "s";
+
+    @Override
+    public List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> listOfAll,
+                                                        List<FruitTransaction> listOfFruits) {
+        for (FruitTransaction listOfFruit : listOfFruits) {
```

Comment:
Using 'listOfFruit' as the name of a single 'FruitTransaction' object in the enhanced for loop is misleading. It should be named to reflect that it is a single transaction, not a list.